CourseDevelopment/Week1/CODE_OF_CONDUCT_template.md

```diff
@@ -0,0 +1,60 @@
+# CODE OF CONDUCT 2022
+
+This code of conduct will been decided by cohort participants and is enacted from DDMMYYYY.
+The procedure is based on [this exercise by Aurelia Moser](http://aureliamoser.com/aaas-guides/conduct/index.html)  
+
+Everyone taking part in the course and discussions (mentors, helpers, coordinators, and learners) is required to conform to the following Code of Conduct. Coordinators will oversee adherence to this code throughout the course.
+ 
+#### Characteristics we value:  
+
+
```

Comment:
Is the idea that the cohort fills out these parts during lecture 1? And do we provide some guidelines/discussion-starters to get the thinking going? If so, explain that here, e.g. in italics?

Reply:
yes, this will happen towards the end of day 1. the instructions are in the website, but does no harm to add them to the template as well. will do.